visualizations/frontend/charts/pie_chart.html

```diff
@@ -85,7 +132,7 @@
         {% if config.hover_mode %}
         "hovermode" : "{{ config.hover_mode }}"
         {% else %}
-        "hovermode" : "x unified"
+        "hovermode" : "false"
```

Comment:
Because false is a boolean it shouldn't be in quotes

Reply:
Oh.. it was by mistake. Thanks for noticing Kennan.